File: app/src/main/java/com/losextraditables/krokodil/core/model/SongParameters.java
package com.losextraditables.krokodil.core.model;

import lombok.Data;

@Data
public class SongParameters {
  public String thumbnailUrl;
  public String duration;
  public String title;
  public String author;
  public String visits;
  public String description;
  public String videoId;
  public Long timesDownloaded;
}

File: app/src/main/java/com/losextraditables/krokodil/core/model/SongParameters.kt
package com.losextraditables.krokodil.core.model

data class SongParameters(val thumbnailUrl: String,
                          val duration: String,
                          val title: String,
                          val author: String,
                          val visits: String,
                          val description: String,
                          val videoId: String,
                          var timesDownloaded: Long) {
}
